<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="SuperDell" name="com.example.tests.ContactRemovalTests" tests="1" failures="1" timestamp="19 Oct 2016 20:13:04 GMT" time="1.872" errors="0">
  <testcase name="deleteSomeContact" time="1.872" classname="com.example.tests.ContactRemovalTests">
    <failure type="java.lang.AssertionError" message="Lists differ at element [1]: DataForCreateContact [namecontact=Дима, familiya=null, town=null, phone1=null, phone2=null, phone3=null, mail1=null, mail2=null, yearBitrh=null, address=null, bilding=null] != DataForCreateContact [namecontact=Дима , familiya=null, town=null, phone1=null, phone2=null, phone3=null, mail1=null, mail2=null, yearBitrh=null, address=null, bilding=null] expected [DataForCreateContact [namecontact=Дима, familiya=null, town=null, phone1=null, phone2=null, phone3=null, mail1=null, mail2=null, yearBitrh=null, address=null, bilding=null]] but found [DataForCreateContact [namecontact=Дима , familiya=null, town=null, phone1=null, phone2=null, phone3=null, mail1=null, mail2=null, yearBitrh=null, address=null, bilding=null]]">
      <![CDATA[java.lang.AssertionError: Lists differ at element [1]: DataForCreateContact [namecontact=Дима, familiya=null, town=null, phone1=null, phone2=null, phone3=null, mail1=null, mail2=null, yearBitrh=null, address=null, bilding=null] != DataForCreateContact [namecontact=Дима , familiya=null, town=null, phone1=null, phone2=null, phone3=null, mail1=null, mail2=null, yearBitrh=null, address=null, bilding=null] expected [DataForCreateContact [namecontact=Дима, familiya=null, town=null, phone1=null, phone2=null, phone3=null, mail1=null, mail2=null, yearBitrh=null, address=null, bilding=null]] but found [DataForCreateContact [namecontact=Дима , familiya=null, town=null, phone1=null, phone2=null, phone3=null, mail1=null, mail2=null, yearBitrh=null, address=null, bilding=null]]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertEquals(Assert.java:125)
	at org.testng.Assert.assertEquals(Assert.java:551)
	at org.testng.Assert.assertEquals(Assert.java:516)
	at com.example.tests.ContactRemovalTests.deleteSomeContact(ContactRemovalTests.java:35)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </failure>
  </testcase> <!-- deleteSomeContact -->
</testsuite> <!-- com.example.tests.ContactRemovalTests -->
